app/src/main/java/com/fstyle/structure_android/screen/main/MainActivity.java

```diff
@@ -99,13 +100,39 @@ protected void onStop() {
 
     @Override
     public void onSearchError(Throwable throwable) {
-        mDialogManager.dialogMainStyle(throwable.getMessage(),
-                new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        dialog.dismiss();
-                    }
-                });
+        mDialogManager.dismissProgressDialog();
+//        mDialogManager.dialogBasicNoTitle("test abc", new MaterialDialog.SingleButtonCallback() {
+//            @Override
+//            public void onClick(@NonNull MaterialDialog materialDialog,
+//                    @NonNull DialogAction dialogAction) {
+//                Log.d(TAG, "onClick: ");
+//            }
+//        });
+//                mDialogManager.dialogBasicCheckPrompt(
+//                        "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+//                        new MaterialDialog.SingleButtonCallback() {
+//                            @Override
+//                            public void onClick(@NonNull MaterialDialog materialDialog,
+//                                    @NonNull DialogAction dialogAction) {
+//                                Log.d(TAG, "onClick: "
+//                                        + materialDialog.isPromptCheckBoxChecked()
+//                                        + " : "
+//                                        + dialogAction.toString());
+//                            }
+//                        });
+                mDialogManager.dialogListMultiChoice("abc", R.array.socialNetworks, new
+         Integer[] { 0, 1 },
+                        new MaterialDialog.ListCallbackMultiChoice() {
+                            @Override
+                            public boolean onSelection(MaterialDialog materialDialog, Integer[]
+         integers,
```

Comment:
ERROR: 'integers' have incorrect indentation level 9, expected level should be 32.